### PR TITLE
filebeat: add docker image name field to journald input

### PR DESCRIPTION
**Pull request by msanft**
## Proposed commit message

Since commit [5f7e102](https://github.com/moby/moby/commit/5f7e102df73672e5b53c813104ecb061c45a5f52) in moby/moby, the `IMAGE_NAME` field is added to Docker log messages sent to journald. This was previously undocumented in Docker, which is changed by [69a143d](https://github.com/docker/docs/pull/20557/commits/69a143dc53a72f1142d2c8fd35fe358a31ddbb88). This adds a translation for the field to Filebeat.

Additionally, the documentation for the `container.log.tag` translation is fixed.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

**Comment by msanft** [comment]
The PR template asks to add labels, but I'm unable to do so.

**Comment by belimawr** [comment]
Thanks for the PR @msanft! It looks great!

I've just merged some changes to the file you edited, I'll merge `main` into your branch to check for any conflicts.

**Review by belimawr — CHANGES_REQUESTED** [review]
The changelog entry is missing, could you add one in `CHANGELOG.next.asciidoc`?

**Comment by msanft** [comment]
Thanks for the review @belimawr!
Added the relevant changelog entry. Feel free to correct me on wording here. I wasn't exactly sure how to put it best.

**Review by belimawr — APPROVED** [review]

**Comment by pierrehilbert** [comment]
run docs-build

**Review by leehinman — APPROVED** [review]

File: filebeat/docs/inputs/input-journald.asciidoc [hunk 1]
```diff
@@ -284,7 +284,8 @@ available:
 `CONTAINER_ID_FULL`::         `container.id`
 `CONTAINER_NAME`::            `container.name`
 `CONTAINER_PARTIAL_MESSAGE`:: `container.partial`
-`CONTAINER_TAG`::             `container.image.tag`
+`CONTAINER_TAG`::             `container.log.tag`
```
Comment (pierrehilbert) [inline filebeat/docs/inputs/input-journald.asciidoc:287]: Just to be sure to understand, what is the reason to switch from `container.image.tag` to `container.log.tag`?

Comment (msanft) [reply filebeat/docs/inputs/input-journald.asciidoc:287]: It's just a matter of correcting the docs. The actual field is called [`container.log.tag`](https://github.com/elastic/beats/blob/4504968401342a289e443e94a2d8547496170bef/filebeat/input/journald/pkg/journalfield/default.go#L80)

Comment (pierrehilbert) [reply filebeat/docs/inputs/input-journald.asciidoc:287]: Make sense thanks for the explanation! 
Looks good to be merged then!

**Pull request merged**